pkgs/development/python-modules/superqt/default.nix

```diff
@@ -7,23 +7,34 @@
 , typing-extensions
 , pytest
 , pytestCheckHook
+, pygments
 }:
 
 buildPythonPackage rec {
   pname = "superqt";
-  version = "0.3.3";
+  version = "0.3.5";
 
   src = fetchFromGitHub {
     owner = "napari";
     repo = pname;
     rev = "refs/tags/v${version}";
-    sha256 = "sha256-Ns3AFUL0BReIwTHfrlfXr/2GLtLvT7hfSjjh+r7btcY=";
+    sha256 = "sha256-nKNFV/mzdugQ+UJ/qB0SkCSm5vEpvI/tgHYKJr6NEyg=";
   };
   format = "pyproject";
+
   nativeBuildInputs = [ setuptools-scm ];
-  propagatedBuildInputs = [ pyqt5 qtpy typing-extensions ];
-  checkInputs = [ pytestCheckHook pytest ];
```

Comment:
Maybe we keep `pytestCheckHook`, that'd remind us to occasionally test if `doCheck = true` still segfaults

Reply:
sure, why not